bin/resources/shaders/dx11/present.fx

```diff
@@ -395,4 +395,43 @@ PS_OUTPUT ps_filter_lottes(PS_INPUT input)
 	return output;
 }
 
+PS_OUTPUT ps_4x_rgss(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	float2 dxy = float2(ddx(input.t.x), ddy(input.t.y));
+	float3 color = 0;
+
+	float s = 1.0/8.0;
+	float l = 3.0/8.0;
+
+	color += sample_c(input.t + float2( s, l) * dxy).rgb;
+	color += sample_c(input.t + float2( l,-s) * dxy).rgb;
+	color += sample_c(input.t + float2(-s,-l) * dxy).rgb;
+	color += sample_c(input.t + float2(-l, s) * dxy).rgb;
+
+	output.c = float4(color * 0.25,1);
+	return output;
+}
+
+PS_OUTPUT ps_automagical_supersampling(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	float2 ratio = (u_source_size / u_target_size) * 0.5;
+	float2 steps = floor(ratio);
+	float3 col = sample_c(input.t).rgb;
+	float div = 1;
+
+	for(float y = 0; y < steps.x; y++)
+		for(float x = 0; x < steps.y; x++) {
```

Comment:
Formatting, above one too, same goes for the other shaders.

Reply:
Brackets and missing space after for.

Reply:
Ooop, one more try then.

Reply:
Done!